Dockerfile

```
@@ -1,5 +1,7 @@
 FROM node:16.3.0-buster-slim
 
+ARG CHROME_PACKAGE_VERSION=95.0.4638.69-1
```

any reason to provide a specific value there?

---

I found it handy if we build the image locally to test changes.

---

Yes it can, but you'll have to remember it each time you do `yarn build` locally.
If you think it's more trouble than practical, I can remove it.

---

```suggestion
ARG CHROME_PACKAGE_VERSION
RUN test -n "$CHROME_PACKAGE_VERSION" || (echo "\nCHROME_PACKAGE_VERSION  not set, you need to run:\ndocker build --build-arg CHROME_PACKAGE_VERSION=xxx\n" && false)
```
with something like that, it would fail fast and avoid to build the image with an outdated version

---

Ok